### DISCUSSION TITLE
Hashcat tutorial abng

### DISCUSSION THREAD

**Discussion by Reyhan-yt2** [dizcza/hashcat-wpa-server#12] [General]
Start tutorial

**Comment by dizcza** [comment]
There are plenty of Hashcat tutorials, for instance, this one: https://www.freecodecamp.org/news/hacking-with-hashcat-a-practical-guide.

If you mean a tutorial on how to use this particular project, it's already written on Readme.

In either case, I do have a tutorial - it's located in `app/templates/learn_more.html` and hosted on my testing website - [Learn More](http://85.217.171.57:9111/learn_more).
